### DISCUSSION TITLE
添加天翼云盘失败

### DISCUSSION THREAD

**Discussion by NekoMirra** [AlistGo/alist#1193] [Q&A]
![image](https://user-images.githubusercontent.com/48759390/171798344-ae4afca7-4539-4652-bfa8-7704c8033144.png)
添加天翼云盘时出现问题，两个版本都无法添加

**Comment by 18600475504** [comment]
我也遇到同样问题，楼主最后解决了吗？

**Comment by NekoMirra** [comment]
以解决,换为桌面板即可&nbsp;


Mirra-塔兔
***@***.***

Mirra来了哦QWQ


&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年8月21日(星期天) 中午11:14
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [alist-org/alist] 添加天翼云盘失败 (Discussion #1193)





 
我也遇到同样问题，楼主最后解决了吗？
